spawn/util/property.py

```diff
@@ -365,30 +365,6 @@ class ArrayProperty(PropertyBase):
     :meth:`__get__`, :meth:`__set__` and :meth:`__delete__` return array wrappers
     that allow indexes to be used
     """
-    def __init__(
-            self, type_, fget=None, fset=None, fdel=None, fvalidate=None,
-            default=None, doc=None, abstract=False, readonly=False):
-        """Initialises :class:`ArrayProperty`
-
-        :param fget: Getter function for property
-        :type fget: func
-        :param fset: Setter function for property
-        :type fset: func
-        :param fdel: Deleter function for property
-        :type fdel: func
-        :param fvalidate: Validation function for property
-        :type fvalidate: func
-        :param default: The default value for this property
-        :type default: object
-        :parm doc: The docstring for this property
-        :type doc: str
-        :param abstract: ``True`` if this property is abstract (requires implementation);
-        ``False`` otherwise.
-        :type abstract: bool
-        """
-        super().__init__(fget, fset, fdel, fvalidate, default, doc, abstract, readonly)
-        self._type = type_
-
```

Comment:
Nope - this has broken array properties. I think pylint told me the `__init__` function was unnecessary but clearly it was wrong!